sds_data_manager/utils/stackbuilder.py

```diff
@@ -18,6 +18,8 @@
     ecr_stack,
     efs_stack,
     ialirt_bucket_stack,
+    ialirt_database_stack,
+    ialirt_ingest_lambda_stack,
```

Comment:
These are coupled together, could we turn this into a single stack?

I still would prefer one "ialirt" stack I think since it is a "deployable unit" all together IMO.

Reply:
An ialirt stack also makes sense to me.